index.js

```diff
@@ -65,6 +66,26 @@ module.exports = class Plugin {
     this.__broccoliFeatures__ = broccoliFeatures;
   }
 
+  get inputPaths() {
```

Comment:
i just realized, likely people are doing unsupported bad things and setting to `this.inputPaths` and `this.outputPaths` if that is the case, we can make a setter to fallback to old functionality or something.

Reply:
Personally I think that we shouldn't support this. These changes will be behind a major version bump and we should not allow that behavior